website/pages/en/developing/substreams-powered-subgraphs-faq.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Substreams-powered subgraphs FAQ
+---
+
+## What are Substreams?
+
+Developed by [StreamingFast](https://www.streamingfast.io/), Substreams is an exceptionally powerful processing engine capable of consuming rich streams of blockchain data. Substreams allow you to refine and shape blockchain data for fast and seamless digestion by end-user applications. More specifically, Substreams is a blockchain-agnostic, parallelized, and streaming-first engine, serving as a blockchain data transformation layer. Powered by the [Firehose](https://firehose.streamingfast.io), it ​​enables developers to write Rust modules, build upon community modules, provide extremely high-performance indexing, and [sink](https://substreams.streamingfast.io/developers-guide/sink-targets#substreams-sinks-overview) their data anywhere.
+
+Go to the [Substreams Documentation](https://substreams.streamingfast.io/) to learn more about Substreams.
+
+## What are Substreams-powered subgraphs?
+
+[Substreams-powered subgraphs](../cookbook/substreams-powered-subgraphs/) combine the power of Substreams with the queryability of subgraphs. When publishing a Substreams-powered Subgraph, the data produced by the Substreams transformations, [formatted in a special way](https://github.com/streamingfast/substreams-sink-entity-changes/blob/develop/substreams-entity-change/src/tables.rs), are inserted into the database as if it was produced by the subgraph itself.
```

Comment:
I suggest avoiding relative links, so that they don't break if this page happened to be moved:

```suggestion
[Substreams-powered subgraphs](/cookbook/substreams-powered-subgraphs/) combine the power of Substreams with the queryability of subgraphs. When publishing a Substreams-powered Subgraph, the data produced by the Substreams transformations, [formatted in a special way](https://github.com/streamingfast/substreams-sink-entity-changes/blob/develop/substreams-entity-change/src/tables.rs), are inserted into the database as if it was produced by the subgraph itself.
```